docs/current/cli/752479-guides.md

```diff
@@ -0,0 +1,7 @@
+---
+slug: /cli/guides
+---
+
+# Guides
+
+- [Proxy Requests to the Dagger API](./guides/632021-proxy-requests-api.md)
```

Comment:
Heads up, we might cut `dagger listen` entirely for security reasons. Discussion underway https://github.com/dagger/dagger/issues/3830#issuecomment-1325629630.

At the very least I would de-prioritize writing this guide versus others.

Reply:
Understood. I'll leave the stubs and navigation in for now, we can finalize this based on the discussion result.

Reply:
Nix, I'm removing the stubs and links to this guide to make this PR ready to merge. I will retain `dagger listen` in the CLI reference for the moment.

Reply:
Fixed and removed dagger listen